apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -120,13 +120,20 @@ export function ConversationView({
         data-qa="conversation-name"
       >
         <Tooltip
-          tooltip={t(
-            getEntityNameError(isNameInvalid, isInvalidPath, isExternal),
-          )}
-          hideTooltip={!isNameOrPathInvalid}
-          triggerClassName="block max-h-5 flex-1 truncate whitespace-pre break-all text-left"
+          tooltip={
+            conversation.name ||
+            t(getEntityNameError(isNameInvalid, isInvalidPath, isExternal))
+          }
+          triggerClassName=" max-h-5 flex-1 truncate whitespace-pre text-left"
+          contentClassName="sm:max-w-[400px] max-w-[250px] break-all"
+          hideTooltip={isNameOrPathInvalid}
```

Comment:
Your can't just remove NOT here, because your hide error always!
Pay attention for what this was done.
Just remove this property to show tooltip always

Reply:
Thanks for your comment, I fixed it.